confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -139,17 +153,22 @@ def register(self, subject, avro_schema):
         schema_id = result['id']
         # cache it
         self._cache_schema(avro_schema, schema_id, subject)
+
         return schema_id
 
-    def get_by_id(self, schema_id):
+    def get_by_id(self, schema_id, json_format=False):
```

Comment:
ideally it would be sufficient, but if fastavro fails for some reason, i wanted to preserve the current consumer functionality where it falls back to to the standard avro library - in which case the schema still needs to be available somehow as a an avro Schema object.